app/views/tasks/_markdown_help_model.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="modal-header">
+  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+  <h1 class="modal-title" id="MarkdownLabel">Синтаксис Markdown</h4>
+</div>
+
+<div class="modal-body">
+  <table class="table table-striped">
+   <thead>
+     <tr>
+       <th><h3>Результат</h3></th>
+       <th><h3>Markdown</h3></th>
+     </tr>
+   </thead>
+   <tbody>
+     <tr>
+       <td><strong>Жирный</strong></td>
+       <td>**Текст**</td>
+     </tr>
+     <tr>
+       <td><em>Курсив</em></td>
+       <td>*Текст*</td>
+     </tr>
+     <tr>
+       <td><a>Ссылка</a></td>
+       <td>[Описание](httl://)</td>
+     </tr>
+     <tr>
+       <td><code>Код</code></td>
+       <td>`код`</td>
+     </tr>
+     <tr>
+       <td>H1</td>
+       <td># Заголовок</td>
+     </tr>
+     <tr>
+       <td>H2</td>
+       <td>## Заголовок</td>
+     </tr>
+     <tr>
+       <td>H3</td>
+       <td>### Заголовок</td>
+     </tr>
+     <tr>
+       <td><ul><li>Маркированный список</li></ol></td>
+       <td>Три пробела + * + пробел</td>
```

Comment:
According to Github's [cheatsheet](https://guides.github.com/features/mastering-markdown/), three spaces are not needed.